src/runner/browser-env/vite/browser-modules/utils/console.ts

```diff
@@ -0,0 +1,16 @@
+import { CONSOLE_METHODS } from "../constants.js";
+import { BrowserEventNames } from "../types.js";
+
+export const wrapConsoleMethods = (): void => {
+    const { socket } = window.__testplane__;
+
+    for (const method of CONSOLE_METHODS) {
+        const origCommand = console[method].bind(console);
+
+        console[method] = (...args: unknown[]): void => {
+            socket.emit(BrowserEventNames.callConsoleMethod, { method, args });
+
+            origCommand(...args);
```

Comment:
nit: I suggest to use `return origCommand(...args)`, just in case if sombody would overwrite console method too

Reply:
I have a synchronous event here. It means I can't return anything to browser. I don't see the point in such an implementation, since the real `console` methods is still running in the browser. Here I just show arguments in terminal.